src/mgmt/cli_mgmt.c

```diff
@@ -507,13 +524,8 @@ int dc_mgmt_net_cfg(const char *name)
 	char                    *cli_srx_set        = NULL;
 	char                    *crt_timeout        = NULL;
 	char                     buf[SYS_INFO_BUF_SIZE];
-	struct dc_mgmt_sys_info  info;
-	Mgmt__GetAttachInfoResp *resp;
-
-	/* Query the agent for the CaRT network configuration parameters */
-	rc = get_attach_info(name, true /* all_ranks */, &info, &resp);
-	if (rc != 0)
-		return rc;
+	struct dc_mgmt_sys_info *info = &info_g;
+	Mgmt__GetAttachInfoResp *resp = resp_g;
```

Comment:
This is just a suggestion, to take or leave if you like. But I think it would be better to use a function to access these globals rather than having multiple places access them directly. I was thinking, you could reduce some of these additional code changes by having `get_attach_info` check the globals first, and only upcall to the agent if they are absent.